swarm_rl/env_wrappers/compatibility.py

```diff
@@ -42,6 +42,9 @@ def step(self, action: Any) -> Tuple[Any, float, bool, bool, Dict]:
             (observation, reward, terminated, truncated, info)
         """
         obs, reward, done, info = self.env.step(action)
+        if isinstance(info, dict):
+            if isinstance(done, bool):
+                done = [done]
```

Comment:
Please add some comments on that, for example, why we need this part of the code. Besides, please use one if clause for that, such as
`if isinstance(info, dict) and isinstance(done, bool)`. And test it if necessary.